### ISSUE TITLE
Perl

### ISSUE THREAD

**Issue by kablamo** [hmarr/codecube#6]
Hello.  Great project!  If I submitted a pull request for Perl support would you accept it?

**Comment by hmarr** [comment]
Absolutely! Give me a shout if you need a hand, although you should only need to touch about 4 lines of code.

**Comment by kablamo** [comment]
I'm a bit stuck.  I can see codecube.io in my browser.  When I try to run some Ruby code nothing happens.  I am running the go 'server' binary.  Its logs say the following:

2013/11/06 07:05:09 New SSE subscriber (WA9pmh)
2013/11/06 07:05:15 Running code for id WA9pmh
2013/11/06 07:05:15 Running code...
2013/11/06 07:05:15 Creating code directory
2013/11/06 07:05:15 Creating source file
2013/11/06 07:05:15 Reserving uid
2013/11/06 07:05:15 Got uid 20000
2013/11/06 07:05:15 Creating container
2013/11/06 07:05:15 [E] Error running code: No such image

But I do have the runner image:

vagrant@precise64:~$ sudo docker images  
REPOSITORY          TAG                 ID                  CREATED             SIZE
runner              latest              82a1c8d61423        10 hours ago        12.29 kB (virtual 614.4 MB)
base                latest              b750fe79269d        7 months ago        24.65 kB (virtual 180.1 MB)
base                ubuntu-12.10        b750fe79269d        7 months ago        24.65 kB (virtual 180.1 MB)
base                ubuntu-quantal      b750fe79269d        7 months ago        24.65 kB (virtual 180.1 MB)
base                ubuntu-quantl       b750fe79269d        7 months ago        24.65 kB (virtual 180.1 MB)

**Comment by kablamo** [comment]
I made some progress!  I had a couple things misconfigured.  I can now get the container to run code.  Now the go server logs are giving me the following error:

2013/11/10 10:18:02 New SSE subscriber (aMB7T2)
2013/11/10 10:18:08 Running code for id aMB7T2
2013/11/10 10:18:08 Running code...
2013/11/10 10:18:08 Creating code directory
2013/11/10 10:18:08 Creating source file
2013/11/10 10:18:08 Reserving uid
2013/11/10 10:18:08 Got uid 20000
2013/11/10 10:18:08 Creating container
2013/11/10 10:18:08 Starting container
2013/11/10 10:18:08 Streaming container logs
2013/11/10 10:18:08 Waiting for container to finish
2013/11/10 10:18:08 New SSE message entrypoint.sh: line 13: /etc/hosts: Read-only file system
2013/11/10 10:18:08 New SSE message hi
2013/11/10 10:18:08 Container exited by itself
2013/11/10 10:18:08 Removing container
2013/11/10 10:18:08 Releasing uid
2013/11/10 10:18:08 Removing code dir
2013/11/10 10:18:08 New SSE message => exited with status 0

Oh hey, so I just commented out line 13 rebuilt the docker image and now everything works!
